packages/react-popover/src/PopoverInternalUpdateContent.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { Popover, PopoverTrigger, PopoverSurface } from './index';
+import { Button } from './utils.stories';
+import { ExampleContent } from './utils.stories';
+
+export const InternalUpdateContent = () => {
+  const [visible, setVisible] = React.useState(true);
+
+  const changeContent = () => setVisible(false);
```

Comment:
IMO there is sense to reset `visible` when popover opens:
- open a popover
- click on a button
- close a popover
- open a popover
- no button?

Reply:
good catch